doc/Oauth2-Configuration.md

```diff
@@ -0,0 +1,32 @@
+# Oauth2 Configuration
+You can use any IDP which supports Oauth2 like [authentik](https://goauthentik.io) for authentification with LibreBooking
+
+## IDP Configuration
```

Comment:
nit: From my quick searching it seems to be "IdP" usually.

Reply:
I changed this too thanks for the feedback